code/modules/mob/living/simple_animal/abnormality/aleph/seasons.dm

```diff
@@ -884,4 +1230,323 @@
+	. = ..()
+	if(args[2])
+		faction = args[2]
+
+/obj/effect/temp_visual/winter_god/Initialize(mapload)
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(Ice_Damage)), 1 SECONDS)
+
+/obj/effect/temp_visual/winter_god/proc/Ice_Damage()
+	addtimer(CALLBACK(src, PROC_REF(Ice_Damage)), 1 SECONDS)
+	for(var/mob/living/L in get_turf(src))
+		if(faction_check(faction, L.faction, FALSE))
+			continue
+		L.apply_damage(5, PALE_DAMAGE, null, L.run_armor_check(null, PALE_DAMAGE), spread_damage = TRUE)
+		if(ishuman(L))
+			var/mob/living/carbon/human/H = L
+			H.become_nearsighted(TRAUMA_TRAIT)
+			addtimer(CALLBACK(src, PROC_REF(Revert),H), 3 SECONDS)
+			if(H.stat >= HARD_CRIT || H.health < 0)
+				if(HAS_TRAIT(H, TRAIT_HUSK))
+					return FALSE
+				var/cube = icon('icons/effects/freeze.dmi', "ice_cube")
+				H.add_overlay(cube)
+				H.adjustBruteLoss(H.maxHealth)
+				H.Drain()
+
+/obj/effect/temp_visual/winter_god/proc/Revert(mob/living/carbon/human/H)
+	H.cure_nearsighted(TRAUMA_TRAIT)
+
+/mob/living/simple_animal/hostile/willo_wisp
+	name = "Willo-O-Wisp"
+	desc = "A blue fire."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "willo_wisp"
+	icon_living = "willo_wisp"
+	icon_dead = "willo_wisp"
+	faction = list("hostile")
+	light_color = COLOR_TEAL
+	light_range = 3
+	light_power = 2
+	is_flying_animal = TRUE
+	density = FALSE
+	speak_emote = list("wispers")
+	attack_verb_continuous = "explodes"
+	attack_verb_simple = "explodes"
+	attack_sound = 'sound/abnormalities/seasons/aoe_attack.ogg'
+	del_on_death = TRUE
+	health = 10
+	maxHealth = 10
+	damage_coeff = list(RED_DAMAGE = 1, WHITE_DAMAGE = 2, BLACK_DAMAGE = 1, PALE_DAMAGE = 1)
+	melee_damage_type = BLACK_DAMAGE
+	melee_damage_lower = 20
+	melee_damage_upper = 20
+	move_to_delay = 1.3 //very fast, very weak.
+	stat_attack = HARD_CRIT
+	ranged = 1
+	retreat_distance = 3
+	minimum_distance = 1
+
+/mob/living/simple_animal/hostile/willo_wisp/Initialize()
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(Boom)), 30 SECONDS)
+
+/mob/living/simple_animal/hostile/willo_wisp/AttackingTarget() //they explode
+	for(var/turf/T in view(2, src))
+		new/obj/effect/season_effect/breath/fall(T)
+		for(var/mob/living/L in T)
+			if(faction_check_mob(L))
+				continue
+			L.apply_damage(50, melee_damage_type, null, L.run_armor_check(null, melee_damage_type), spread_damage = TRUE)
+	qdel(src)
+	return ..()
+
+/mob/living/simple_animal/hostile/willo_wisp/death(gibbed)
+	Boom()
+	..()
+
+/mob/living/simple_animal/hostile/willo_wisp/proc/Boom(gibbed)
+	for(var/turf/T in view(1, src))
+		new/obj/effect/season_effect/breath/fall(T)
+		for(var/mob/living/L in T)
+			if(faction_check_mob(L))
+				continue
+			L.apply_damage(30, melee_damage_type, null, L.run_armor_check(null, melee_damage_type), spread_damage = TRUE)
+	qdel(src)
+
+//spring mobs
+
+/mob/living/simple_animal/hostile/flora_zombie
+	name = "Flora Zombie"
+	desc = "What appears to be human, only overgrown and decayed..."
```

Comment:
This could use a more apt description, humans are not plants - they cannot be overgrown.